test/test_simulation_build.jl

```diff
@@ -1,13 +1,15 @@
-path = joinpath(pwd(), "test_sequence_build")
-!isdir(path) && mkdir(path)
+g_test_path = joinpath(pwd(), "test_sequence_build")
```

Comment:
The changes in this file are only there to prevent stages from being rebuilt.  They can be reverted if we don't go with that change.  I would argue that this approach is better regardless.